gpmp2/kinematics/PriorWorkspacePoseVelocity.h

```diff
@@ -0,0 +1,125 @@
+/**
+ *  @file   PriorWorkspacePoseVelocity.h
+ *  @brief  Prior defined on the workspace pose and velocity of final joint
+ *(end-effector) of a robot given its state in configuration space
+ *  @author Matthew King-Smith
+ *  @date   May 16, 2023
+ **/
+
+#pragma once
+
+#include <gpmp2/kinematics/Arm.h>
+#include <gtsam/base/Matrix.h>
+#include <gtsam/base/Vector.h>
+#include <gtsam/geometry/Pose3.h>
+#include <gtsam/nonlinear/NonlinearFactor.h>
+
+#include <iostream>
+#include <vector>
+
+namespace gpmp2 {
+
+/**
+ * binary factor Gaussian prior defined on the workspace pose and velocities
+ */
+class PriorWorkspacePoseVelocity
```

Comment:
To be consistent with GTSAM's naming scheme, this should probably be called WorkspacePoseVelocityPrior

Reply:
Fixed.